packages/collector/test/test_util/ProcessControls.js

```diff
@@ -169,7 +169,8 @@ class ProcessControls {
         () =>
           this.sendRequest({
             method: 'GET',
-            suppressTracing: true
+            suppressTracing: true,
+            checkStatusCode: true
```

Comment:
This is the fix for respecting the response from the app for the health endpoint `/`.